test/units/galaxy/test_collection.py

```diff
@@ -697,7 +697,7 @@ def test_build_ignore_older_release_in_root(collection_input, monkeypatch):
 
 
 def test_build_ignore_patterns(collection_input, monkeypatch):
-    input_dir = collection_input[0]
+    input_dir = os.path.join(collection_input[0], '')
```

Comment:
This should be enough to enforce the test of this bugfix, this will effectively fail with the current code in the devel branch. @webknjaz there are paths concatenations everywhere which might be prune to failures, creating a test for this should be easy but making sure we normalize paths should be simpler than creating extra unit tests. WDYT? Again thanks for the quick feedback!

Reply:
Yeah, I thought about parametrizing the collection path but I wanted to keep the change as small as possible xD this should now do the trick, the only thing is I had to make the path configurable when initializing the collection.

Reply:
Marking this thread as resolved.